docs/html/reference/pip_install.rst

```diff
@@ -402,28 +402,30 @@ Git
 ^^^
 
 pip currently supports cloning over ``git``, ``git+http``, ``git+https``,
-``git+ssh``, ``git+git`` and ``git+file``:
+``git+ssh``, ``git+git`` and ``git+file``,  but note that the ``git``, ``git+git``,
+and ``git+http`` are not recommended due to their lack of security.
+(The former two uses `the Git Protocol.`_)
```

Comment:
nit: language and period outside link (so that the text can be re-arranged around without needing to update the link-reference below)

```suggestion
(The former two use `the Git Protocol`_.)
```

Reply:
Fixed